recipes/libtheora/meta.yaml

```diff
@@ -44,6 +44,7 @@ test:
 
 about:
   home: https://theora.org
+  license: BSD-like
```

Comment:
If you look at their repository at https://github.com/xiph/theora, you see Github recognizes the license as BSD-3-Clause (top right badge in the stats bar), so I would just leave it at that.